### ISSUE TITLE
RuntimeWarning running pre-build with python 3.8

### ISSUE THREAD

**Issue by leviathan747** [xtuml/pyxtuml#23]
In later versions of Python I am getting this warning when using the pre-builder:

```
/Users/levi/.pyenv/versions/3.8.2/lib/python3.8/runpy.py:126: RuntimeWarning: 'bridgepoint.prebuild' found in sys.modules after import of package 'bridgepoint', but prior to execution of 'bridgepoint.prebuild'; this may result in unpredictable behaviour
  warn(RuntimeWarning(msg))
```

I found [this StackOverflow article](https://stackoverflow.com/questions/43393764/python-3-6-project-structure-leads-to-runtimewarning)

I was able to get rid of the warnings by removing these two lines from `bridgepoint/__init__.py` (as suggested in the article):

```
from .prebuild import prebuild_action
from .prebuild import prebuild_model
```

but I'm not sure if that will break the API for other users. It passed all the unit tests with python 3.8 on my machine.

**Comment by leviathan747** [comment]
I will also note that I have not noticed any wrong behavior -- just the annoying warning.

**Comment by john-tornblom** [comment]
I've seen this too, thank you for filing an issue. I am also unsure on what the correct course of action is here, I'll do some digging. In the meantime, any insightful feedback is welcome!

**Comment by john-tornblom** [comment]
Fixing this issue without breaking API will be somewhat unpleasant (do-able, but ugly). I suggest we disable the warning in version 2.x as this warning is a false alarm, and add new invokable modules for command line interfaces to develop/3.0, e.g., "python -m xtuml.consistency_check --> python -m xtuml.consistency_checker". Affected CLI commands are:

- xtuml.consistency_check  --> xtuml.consistency_checker
- bridgepoint.consistency_check --> bridgepoint.consistency_checker
- bridgepoint.interpret --> bridgepoint.interpreter
- bridgepoint.oal --> bridgepoint.oal_tree_dumper
- bridgepoint.prebuild --> bridgepoint.prebuilder

Sounds OK?

**Comment by leviathan747** [comment]
Yes, this sounds like a reasonable move forward. I agree based on what I read that there is not a good way to fix it in v2.

**Comment by john-tornblom** [comment]
fixed in dfcab157616f496fe822b398c86069ab6db86f70 and 74f5ca8590cba2f78b1c70a3bcb6accc6ad53228
